api/queries_issue.go

```diff
@@ -608,6 +609,129 @@ func IssueUpdate(client *Client, repo ghrepo.Interface, params githubv4.UpdateIs
 	return err
 }
 
+func IssueTransfer(client *Client, destRepo *Repository, issue Issue) (string, error) {
```

Comment:
We are trying not to expand the `api` package with extra specialized functionality anymore. You can move this to under the new `issue/transfer` package. That way you also don't have to export it anymore.